pypeit/wavemodel.py

```diff
@@ -716,8 +716,8 @@ def create_linelist(wavelength, spec, fwhm, sigdetec=2.,
 
 
 def create_OHlinelist(resolution, waveminmax=(0.8,2.6), dlam=40.0, flgd=True, nirsky_outfile=None,
-                      fwhm=None, sigdetec=3., line_name='OH', file_root_name=None, iraf_frmt=False, 
-                      debug=False):
+                      fwhm=None, sigdetec=3., line_name='OH', file_root_name=None, iraf_frmt=False,
```

Comment:
convert_air_to_vac should never be an option. I think you misunderstood what I wanted here. Happy to chat more. The vacuum option in create_linelist should be renamed to convert_air_to_fac. In the code to create linelists, there should be no convert_air_to_vac option anymore since all the input linelists (Rousselot, HITRAN) are in vacuum.